azure-pipelines/get-tool-versions.yml

```diff
@@ -0,0 +1,44 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+trigger: none
+pr: none
+schedules:
+- cron: "0 3 * * *"
+  displayName: First daily build
+  branches:
+    include:
+    - master
+  always: true
+- cron: "0 15 * * *"
+  displayName: Second daily build
+  branches:
+    include:
+    - master
+  always: true
+
+stages:
+- stage: Get_New_Versions
+  dependsOn: []
+  jobs:
+  - job: Get_Tool_Versions
+    pool:
```

Comment:
We can move `Pool` and pool `name` to global variables.

Reply:
Thanks